sdk/core/azure-core/pom.xml

```diff
@@ -277,5 +279,44 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>native-linux</id>
+      <activation>
+        <property>
+          <name>native-linux</name>
```

Comment:
Should the CI pipelines be updated to activate these profiles?

Reply:
CI pipeline could leverage this. Another option would change the profile to activate based on the OS, but that makes it impossible to run a cross-OS compilation (ex. Linux build system for Windows JAR)